README.md

```diff
@@ -180,3 +196,32 @@ scripts if requested.
 
 There are several tools trying to solve similar problems as Jujutsu. See
 [related work](https://martinvonz.github.io/jj/latest/related-work) for details.
+
+## Contributing
+
+We welcome outside contributions, and there's plenty of things to do, so
+don't be shy. Please ask if you want a pointer on something you can help with,
+and hopefully we can all figure something out.
+
+We do have [a few policies and
+suggestions](https://martinvonz.github.io/jj/prerelease/contributing/)
+for contributors. The broad TL;DR:
+
+- Bug reports are very welcome!
+- Every single patch is code reviewed.
```

Comment:
Maybe "Every individual commit is code reviewed" to try to make it clearer that this is not equivalent to "Every single PR is code reviewed"?